### PR TITLE
Adds softmax kernel

### PR DESCRIPTION
**Pull request by save-buffer**

### CODE REVIEW

File: libraries/bsg_manycore_responder.h [hunk 1]
```diff
@@ -150,10 +150,11 @@ int hb_mc_responder_del(hb_mc_responder_t *responder);
...
 			~__bsg_manycore_ ## rspdr ## _add_del () {	\
-				int r = hb_mc_responder_del(&rspdr);	\
-				if (r != HB_MC_SUCCESS)			\
-					throw r;			\
-			}						\
+                                int r = hb_mc_responder_del(&rspdr);    \
+                                if (r != HB_MC_SUCCESS)                                       \
+                                        bsg_pr_err("%s: failed to remove responder %s: %s\n", \
+                                                    __FILE__, #rspdr, hb_mc_strerror(r));      \
+			}					\
```
Comment (drichmond) [inline libraries/bsg_manycore_responder.h]: Can you file a separate PR for this?

Comment (drichmond) [reply libraries/bsg_manycore_responder.h]: This is awesome, but not really part of the PR

Comment (save-buffer) [reply libraries/bsg_manycore_responder.h]: OK

Comment (drichmond) [reply libraries/bsg_manycore_responder.h]: You might need to rebase on master

Comment (save-buffer) [reply libraries/bsg_manycore_responder.h]: OK, rebased!

**Pull request merged**